virt-who-config.5

```diff
@@ -2,10 +2,12 @@
 .SH NAME
 virt-who-config - configuration for virt-who
 .SH SYNOPISIS
+/etc/sysconfig/virt-who
 /etc/virt-who.conf
 /etc/virt-who.d/*
 .SH DESCRIPTION
-Configuration format is ini-like.
+Configuration format is ini-like for files /etc/virt-who.conf and /etc/virt-who.d/*.
+The contents of /etc/sysconfig/virt-who are environment variables to be used when virt-who is run as a service.
```

Comment:
When run as a service the variables in /etc/sysconfig/virt-who are added to the environment of the virt-who process. This is yet another means of configuring virt-who.